java/src/jmri/jmrit/display/layoutEditor/LayoutBlock.java

```diff
@@ -68,23 +70,18 @@
  */
 public class LayoutBlock extends AbstractNamedBean implements PropertyChangeListener {
 
-    private final boolean enableAddRouteLogging = false;
-    private final boolean enableUpdateRouteLogging = false;
-    private boolean enableDeleteRouteLogging = false;
-    private final boolean enableSearchRouteLogging = false;
-
     private static final List<Integer> updateReferences = new ArrayList<>(500);
 
     // might want to use the jmri ordered HashMap, so that we can add at the top
     // and remove at the bottom.
     private final List<Integer> actedUponUpdates = new ArrayList<>(500);
 
+    @Deprecated (since="5.11.2",forRemoval=true) // please use the SLF4J categories.
     public void enableDeleteRouteLog() {
-        enableDeleteRouteLogging = false;
```

Comment:
Would it make sense to have a `jmri.util.LoggingUtil.warnOnce(..)` here in case somebody has been using this e.g. from scripts?  That's not a very likely situation, I guess. (Two places)

Reply:
Thanks for looking over @bobjacobsen ,
no harm in adding the warnOnce.